osm2streets/src/road.rs

```diff
@@ -54,6 +54,42 @@ pub struct Road {
     pub complicated_turn_restrictions: Vec<(RoadID, RoadID)>,
 
     pub lane_specs_ltr: Vec<LaneSpec>,
+
+    pub stop_line_start: StopLine,
+    pub stop_line_end: StopLine,
+}
+
+#[derive(Clone, Debug, PartialEq, Serialize, Deserialize)]
+pub struct StopLine {
+    /// Relative to the road's reference_line. Stop lines at the start of the road will have low
+    /// values, and at the end will have values closer to the reference_line's length. This is only
+    /// set when the stop line is explicitly specified; it's never inferred.
+    pub vehicle_distance: Option<Distance>,
```

Comment:
Note the big change here. By default roads have no vehicle or bike stop line, only setting it if explicitly found something in OSM